keps/sig-api-machinery/2155-clientgo-apply/kep.yaml

```diff
@@ -17,17 +17,17 @@ see-also:
   - "/keps/sig-api-machinery/0006-apply.md"
 
 # The target maturity stage in the current dev cycle for this KEP.
-# stage: alpha
+stage: beta
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-# latest-milestone: "v1.21"
+latest-milestone: "v1.21"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
-# milestone:
-#  alpha: "v1.21"
-#  beta: "v1.21"
+milestone:
+  alpha: "v1.21"
+  beta: "v1.21"
```

Comment:
Any particular reason we're skipping alpha on this one?

Reply:
Hi, 1.21 Enhancements lead here. Graduation criteria state that it's going straight to `stable`. If that's the case, can you make sure that it's reflected here as well. If it's going to be `beta` or `alpha`, please update the graduation criteria to match what's written here.

Reply:
See https://github.com/kubernetes/enhancements/pull/2419#issuecomment-773739301. The basic thought is- since we don't have feature gates in client go, this functionality is effectively gated only by the Server Side Apply feature in the apiserver, which does have a feature gate, and which is in beta.